tests/sentry/api/serializers/test_organization.py

```diff
@@ -36,8 +36,6 @@ def test_simple(self):
                 "sso-basic",
                 "symbol-sources",
                 "custom-symbol-sources",
-                "tweak-grouping-config",
-                "grouping-info",
```

Comment:
Removing the server feature flags in the same change as removing the frontend checks can result in people 'losing features' whilst they have old client code running and the server is deployed. That scenario might not be super important as grouping configuration changes aren't a 'hot path' but something to keep in mind for other feature flag cleanups.